unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/DefaultChatEntry.cs

```diff
@@ -186,11 +186,14 @@ public void OnPointerClick(PointerEventData pointerEventData)
             int linkIndex = TMP_TextUtilities.FindIntersectingLink(body, pointerEventData.position, DataStore.i.camera.hudsCamera.Get());
             if (linkIndex != -1)
             {
-                DataStore.i.HUDs.gotoPanelVisible.Set(true);
                 TMP_LinkInfo linkInfo = body.textInfo.linkInfo[linkIndex];
-                ParcelCoordinates parcelCoordinate =
-                    CoordinateUtils.ParseCoordinatesString(linkInfo.GetLinkID().ToString());
-                DataStore.i.HUDs.gotoPanelCoordinates.Set(parcelCoordinate);
+                if (!ChannelUtils.IsAChannel(linkInfo.GetLinkID()))
```

Comment:
No, it is intentional. In this case, in order to avoid conflicts between coordinate/channels links detection, we ensure the text is not a channel link.

It will change when we move the coordinate links detection logic from the chat entry to an independent component as we are already doing with channels.